lib/components/Slider/Slider.test.tsx

```diff
@@ -25,17 +12,56 @@ describe('Slider', () => {
     const onValueChangeMock = vi.fn();
     const user = userEvent.setup();
 
-    render(<Slider label="Target" onValueChange={onValueChangeMock} value={0} />);
+    act(() => {
+      render(<Slider label="Target" onValueChange={onValueChangeMock} value={0} rightLabel="moi" />);
+    });
 
     const [sliderThumb] = screen.getAllByRole('slider', { hidden: true });
+
     await waitFor(() => sliderThumb.focus());
+    expect(sliderThumb).toHaveFocus();
+    const tooltip = screen.getByRole('tooltip');
+    expect(tooltip).toHaveTextContent('100 - 0 %');
+
+    const press = async (key: 'left' | 'right') => await user.keyboard(key === 'left' ? '[ArrowLeft]' : '[ArrowRight]');
+    const ariaValue = 'aria-valuenow';
+
+    expect(sliderThumb).toHaveAttribute(ariaValue, '0');
+
+    await press('right');
+    expect(onValueChangeMock).toHaveBeenCalledWith(25);
+    await press('right'); // For some reason, the "aria-valuenow" attribute is not updated without extra right press
```

Comment:
This part drove me nuts. Even though the `onValueChange` is called with the right value, the `aria-valuenow` attribute would not update without double press here, no matter what I tried. When testing with storybook, the keyboard controls work fine, so I think the issue is within the test environment.
Just in case, I tested all the available values.

Things I tried:
* Adding delays here and there
* Using `act` everywhere
* Using `fireEvent` instead of `userEvent`
* Updating ARK UI to 3.X.X
* Updating `@testing-library/*` packages